src/stories/Dialog.stories.mdx

```diff
@@ -241,7 +241,20 @@ An example with a long title that wraps onto the next line.
     name="listbox inside dialog"
     args={{
       title: '3D drawing style',
-      slots: `<template #body><AListbox model-value="" :options="[{ value: 0, label: 'Graphic' }, { value: 1, label: 'Rendered' }, { value: 2, label: 'Black and white' }]" /></template>`
+      slots: `<template #body><AListbox escape-overflow model-value="" :options="[{ value: 0, label: 'Graphic' }, { value: 1, label: 'Rendered' }, { value: 2, label: 'Black and white' }]" /></template>`
```

Comment:
I tried to find a way to open the listbox for Chromatic UI snapshots, but failed. So it's as before (so not worse) but the fact that we don't open the listbox is what made us not realise that https://github.com/archilogic-com/ui-components/pull/330 introduced a regression...

Reply:
fixed, by not using a dialog...